lib/nx_deflate.c

```diff
@@ -2231,27 +2232,39 @@ int deflateInit2_(z_streamp strm, int level, int method, int windowBits,
 			prt_info("this stream has been initialized by sw\n");
 		}
 
-		rc = nx_deflateInit2_(strm, level, method, windowBits, memLevel, strategy, version, stream_size);
-		if(rc != Z_OK){
+		rc = nx_deflate_init(strm, level, method, windowBits, memLevel,
```

Comment:
Where is `nx_deflate_init` defined?

Reply:
It was part of another patch that I did not include in this first draft version. I'll fix that on the next push.